src/devices/Ssd1306/tests/Command/ActivateScrollTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Ssd1306.Command;
+using Xunit;
+
+namespace Iot.Device.Mcp23xxx.Tests
+{
+    public class ActivateScrollTests
+    {
+        [Theory]
+        [InlineData(new byte[] { 0x2F })]
```

Comment:
NIT: Are you expecting to send more data here? if not, this could also be a Fact instead.

Reply:
same comment on all other one-case theories.